kiwi/schema/kiwi.rnc

```diff
@@ -3516,6 +3565,7 @@ div {
             k.namedCollection* &
             k.collectionModule* &
             k.product* &
+            k.container* &
```

Comment:
As I wrote in my long first comment I think we can make our live a bit easier if we just add a new container element inside of the ```<packages>``` section. A packages sub-element in kiwi speak can be a package, an archive and now also a container. The registry information should be imho placed in an URI reference which is the way we support source references in kiwi. I don't think we need to invent a different way to do this unless I forgot something ?